include/clang/AST/Expr.h

```diff
@@ -4748,27 +4752,68 @@ class CountBoundsExpr : public BoundsExpr {
   }
 };
 
+class RelativeBoundsExpr : public BoundsExpr {
```

Comment:
I see. I will try to make relative class as a helper class and write a comment about this class.